### ISSUE TITLE
0 lr: Zero learning rate during the entire training

### ISSUE THREAD

**Issue by zshn25** [facebookresearch/dinov2#320]
Hi tried to run the training on local machine with single GPU with `dinov2/train/train.py` and since the beginning the learning rate was set to 0 and did not change. My configs were default `vitl14.yaml`

**Comment by qasfb** [comment]
There is a learning rate scaling logic here https://github.com/facebookresearch/dinov2/blob/main/dinov2/utils/config.py#L21
The zero seems to happen because of the small global batch size, and because the print only shows 5 digits.
The code is not expected to provide good results with this config using 1 GPU, for info...
